README.md

```diff
@@ -54,7 +89,7 @@ d.rax = 0
 ```
 
 ## Memory Access
-`memory` is used to access the memory of the debugged program. You can use the `d.memory` property to read from and write to it.
+The debugger property `memory` is used to read and write a memory address or range in the virtual memory of the debugged program.
```

Comment:
Either "read to and write from a memory address" or "read and write a range of data starting from a memory address"